wiki/Community/Forum/Abandoned_Wasteland/en.md

```diff
@@ -5,8 +5,8 @@ tags:
   - wastelanded
 ---
 
-# Abandoned wasteland
+# Abandoned Wasteland
 
-The **abandoned wasteland** is a private [subforum](https://osu.ppy.sh/community/forums)<!-- TODO --> used to hide forums<!-- TODO: link --> and forum topics<!-- TODO: link --> from public view. This subforum is only accessible to [moderators](/wiki/People/The_Team/Global_Moderation_Team) who can modify the forums. The instructions to moderators concerning it are: "Put stuff we do not really need in here."
+The **Abandoned Wasteland** is a private [subforum](/wiki/Community/Forum) used to hide [forums](/wiki/Community/Forum) and [forum topics](/wiki/Community/Forum#special-forums) from public view. This subforum is only accessible to [moderators](/wiki/People/The_Team/Global_Moderation_Team) who can modify the forums. The instructions to moderators concerning it are: "Put stuff we do not really need in here."
```

Comment:
the 3rd link is wrong, and only one link needed for the term (article contains all of them)

```suggestion
The **Abandoned Wasteland** is a private subforum used to hide [forums](/wiki/Community/Forum) and forum topics from public view. This subforum is only accessible to [moderators](/wiki/People/The_Team/Global_Moderation_Team) who can modify the forums. The instructions to moderators concerning it are: "Put stuff we do not really need in here."
```